rotkehlchen/chain/ethereum/modules/nfts.py

```diff
@@ -134,10 +135,11 @@ def get_balances(
         # Be sure that the only addresses queried already exist in the database. Fix for #4456
         queried_addresses = list(set(accounts.eth) & set(addresses))
         result: DefaultDict[ChecksumEvmAddress, List[Dict[str, Any]]] = defaultdict(list)
-        nft_results, _ = self._get_all_nft_data(queried_addresses, ignore_cache=ignore_cache)
+        _nft_results, _ = self._get_all_nft_data(queried_addresses, ignore_cache=ignore_cache)
+        nft_results = self._filter_ignore_nfts(_nft_results)
         cached_db_result = self.get_nfts_with_price()
         cached_db_prices = {x['asset']: x for x in cached_db_result}
-        db_data: List[Tuple[str, str, Optional[str], Optional[str], int, ChecksumEvmAddress]] = []
+        db_data: List[Tuple[str, Optional[str], Optional[str], Optional[str], int, ChecksumEvmAddress]] = []  # noqa: E501
```

Comment:
why did the type here changed?

Reply:
`NFT` has `name` as optional but `db_data` defines it as non-optional. Strange mypy didn't catch it